backend/app/models/article.rb

```diff
@@ -2,6 +2,7 @@ class Article < ActiveRecord::Base
   include Utils
 
   before_save :published_post
+  after_save :reading_time
```

Comment:
Use before_save instead to avoid multiple queries. And just set the column reading_time instead of using update_column. Exactly like published_post.

Also, use a different name for the method from the column, maybe calculate_reading_time or set_reading_time